vars/influxWriteData.groovy

```diff
@@ -72,13 +72,18 @@ InfluxDB data map: ${script.commonPipelineEnvironment.getInfluxCustomDataMap()}
 
 private void writeToInflux(config, script){
     if (config.influxServer) {
-        step([
-            $class: 'InfluxDbPublisher',
-            selectedTarget: config.influxServer,
-            customPrefix: config.influxPrefix,
-            customData: script.commonPipelineEnvironment.getInfluxCustomData(),
-            customDataMap: script.commonPipelineEnvironment.getInfluxCustomDataMap()
-        ])
+        try {
+            step([
+                $class: 'InfluxDbPublisher',
+                selectedTarget: config.influxServer,
+                customPrefix: config.influxPrefix,
+                customData: script.commonPipelineEnvironment.getInfluxCustomData(),
+                customDataMap: script.commonPipelineEnvironment.getInfluxCustomDataMap()
+            ])
+        } catch (NullPointerException e){
```

Comment:
In case the NPE contains a reasonable message we could forward that message to the message of the new exception.

Reply:
What about catching only NPEs without messages?

Reply:
technically possible, but a rather unusual pattern ...

What I have in mind is:

```
} catch(NullPointerException e) {
error "[$STEP_NAME] ${e.getClass.getName()} with message ${e.getMessage() ?: '<n/a>'} occured. Is the correct target defined?"
} 
```

```${e.getClass().getName()}``` since subclasses are also caught. Not that likely in case of NPE, but in general helpful in case there is an error reported (also having the message (if any) makes life easier in case of a support case).

But the PR is already approved (also by me). So feal free to merge like if you think it is over-engineered. ...